include/seqan3/alignment/pairwise/execution/execution_handler_sequential.hpp

```diff
@@ -59,7 +60,36 @@ struct execution_handler_sequential
         static_assert(std::ranges::view<first_range_type>, "Expected a view!");
         static_assert(std::ranges::view<second_range_type>, "Expected a view!");
 
-        delegate(func(idx, std::move(first_range), std::move(second_range)));
+        delegate(algorithm(idx, std::move(first_range), std::move(second_range)));
+    }
+
+    /*!\brief Takes underlying range of sequence pairs and invokes an alignment on each instance.
+     * \tparam fn_type           A function which performs the alignment.
```

Comment:
You applied my suggestion to the wrong line... I didn't mean to change the \tparam, but rather the \param below!

Reply:
ups